### PR TITLE
add Dependabot

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
Ready

## Related Issues
fixes: 

## Description
Added Dependabot

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Review by tkatzir — APPROVED** [review]
Great! Approved.
Please wait for @DeanArbel 's approval as well.

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**
